src/clj/fluree/db/json_ld/migrate/sid.cljc

```diff
@@ -127,5 +126,5 @@
                                 async/merge
                                 (async/pipe changes-ch))
           db                (<? (merge-commits ledger commit-opts* tuples-chans))]
-      (ledger/-db-update ledger db)
+      (jld-ledger/db-update ledger db)
```

Comment:
Was the idea here to support other implementations of the ledger protocol? Any reason to try to preserve that abstraction?

Reply:
I'm not concerned with that in this migration ns because this should be one-time use code to bring existing data to the current (as of April 2024) data format, which is a state where multiple versions of the ledger protocol are still not supported.

Long term though, I think it could very well be the case that we have multiple implementations of the ledger protocol (it's also possible that it won't be necessary, i'm not sure). Even if we do have multiple implementations though, I doubt that we'd want a `db-update` method on that protocol as a ledger's latest db is an internal implementation detail about it's own state that shouldn't be manipulated from the outside. The ledger protocol would have a method like `commit!` or `transact!` instead, and a `db-update` function would be internal to a particular ledger and would be called in the implementation of `commit!` or `transact!` instead. That's why I removed that method from the protocol in this branch. 

Of course, there's still work left to do in streamlining the protocols; this is just a start.